web/client/reducers/__tests__/mediaEditor-test.js

```diff
@@ -113,4 +113,29 @@ describe('Test the mediaEditor reducer', () => {
         expect(state.open).toEqual(open);
         expect(state.owner).toEqual(owner);
     });
+    it('UPDATE_ITEM', () => {
+        const map = {
+            id: "resId",
+            layers: [{id: "layerId"}]
+        };
+
+        let state = mediaEditor({
+            settings: {
+                mediaType: "map",
+                sourceId: "geostoreMap"
+            },
+            data: {
+                map: {
+                    geostoreMap: {
+                        resultData: {
+                            resources: [{
+                                id: "resId"
+                            }]
+                        }
+                    }
+                }
+            }
+        }, updateItem({map}));
```

Comment:
to be coherent with the action interface
```suggestion
        }, updateItem(map));
```